docs/napari-plugin-guide.md

```diff
@@ -10,120 +11,163 @@ Launch `napari` with `recOrder`
+![](./images/cap_bg.png)
 
-![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/bg_example.png)
+It is normal to see background retardance and orientation. We will use these background images to correct the data we collect our acquisitions of the sample. 
 
 ### Advanced Tab
 The advanced tab gives the user a log output which can be useful for debugging purposes. There is a log level “debugging” which serves as a verbose output. Look here for any hints as to what may have gone wrong during calibration or acquisition.
 
-![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/advanced.png)
+## Acquisition / Reconstruction Tab
+This acquisition tab is designed to acquire and reconstruct single volumes of both phase and birefringence measurements to allow the user to test their calibration and background. We recommend this tab for quick testing and the Micromanager MDA acquisition for high-throughput data collection.
 
-### Example of Successful Calibration
-Below is an example of a successful calibration with a reasonable extinction value:
+### Acquire Buttons
+![](./images/acquire_buttons.png)
 
-![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/calib_finished.png)
+The **Retardance + Orientation**, **Phase From BF**, and **Retardance + Orientation + Phase** buttons set off MicroManager acquisitions that use the upcoming acquisition settings. After the acquisition is complete, these routines will set off `recOrder` reconstructions that estimate the named parameters. 
 
-## Acquisition / Reconstruction Tab
-This acquisition module is designed to take single image volumes for both phase and birefringence measurements and allows the user to test the outcome of their calibration with small snapshots or volumes. We recommend this acquisition mode for quick testing and the Micromanager MDA acquisition for high-throughput data collection.
-
-### Save Path
-User specifies the directory in which the images will be saved.
+The **STOP** button will end the acquisition as soon as possible, though MicroManager acquisitions cannot always be interrupted. 
 
 ### Acquisition Settings
-*Z Start, Z End, Z Step* specify the relative z-parameters to use for acquiring an image volume. Values are in the default units of the stage, typically in microns. The center slice will be the current position of the z-stage
+![](./images/acquisition_settings.png)
+
+The **Acquisition Mode** sets the target dimensions for the reconstruction. Perhaps surprisingly, all 2D reconstructions require 3D data except for **Retardance + Orientation** in **2D Acquisition Mode**. The following table summarizes the data that will be acquired when an acquisition button is pressed in **2D** and **3D** acquisition modes:
 
-![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/acquisition.png)
+| **Acquisition** \ Acquisition Mode | 2D mode | 3D mode |  
+| :--- | :--- | :--- |
+| **Retardance + Orientation** | CYX data | CYX data | 
```

Comment:
The previous paragraph says:

> The following table summarizes the data that will be acquired

So in the case of 3D birefringence, although the reconstruction only sees a CYX at a time, CZYX will still be acquired (and reconstructed).

Reply:
That was an error! Good eye. Retardance + Orientation 3D should be CZYX.

Reply:
Fixed